README.md

```diff
@@ -19,15 +24,103 @@ Resources are checked on an opt-in model, meaning if you
 provide any app in the `config.yaml`, then all deployed apps must match the allow
 list.
 
+## Running Watchtower
+Watchtower can be run from anywhere that is able to hit your cloud foundry api.
+To run, either download a pre-compiled binary from the [releases](https://github.com/18F/watchtower/releases)
+page, or compile the go source yourself using `go build`.
+
+### CLI Arguments
+
+| Argument | Description |
+| --- | --- |
+| `-config` | Path to the configuration file. Default is "config.yaml" in the current directory |
```

Comment:
single '-' ?

Reply:
Yes, a single dash is the correct form. See https://pkg.go.dev/flag#hdr-Command_line_flag_syntax

Reply:
I see Go thinks it is special. Well at least: "One or two minus signs may be used; they are equivalent."